packages/shared/forks/invokeGuardedCallbackImpl.www.js

```diff
@@ -3,6 +3,8 @@
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
+ *
+ * @noflow
```

Comment:
These files contain flow syntax but are not flow checked. `@noflow` explicitly opts these files into the transform to allow the flow syntax. Imho, we should probably clean this up to either make them truly type checked or remove the type annotation syntax (move it to a comment or something that's not misleading).